qds_sdk/commands.py

```diff
@@ -228,9 +228,9 @@ def get_results(self, fp=sys.stdout, inline=True, delim=None, fetch=True):
         else:
             if fetch:
                 storage_credentials = conn.get(Account.credentials_rest_entity_path)
-                boto_conn = boto.connect_s3(aws_access_key_id=storage_credentials['storage_access_key'],
-                                            aws_secret_access_key=storage_credentials['storage_secret_key'],
-                                            security_token = storage_credentials['session_token'])
+                boto_conn = boto.connect_s3(aws_access_key_id=storage_credentials.get('storage_config').get('access_key'),
+                                            aws_secret_access_key=storage_credentials.get('storage_config').get('secret_key'),
+                                            security_token = storage_credentials.get('storage_config').get('session_token'))
```

Comment:
Its confusing that we are using new v2 response of api but we are supporting only v1.2 as version. But since we don't support multiple version in account , this looks fine. We can support multiple version change later.